README.md

```diff
@@ -2,36 +2,74 @@
 
 ![][status]
 
-The [HACL*] repository is a collection of high-assurance cryptographic algorithms developed as part of [Project Everest].
-It includes source code written in [F*], generated code in C, verified assembly code
-from the [Vale] project, and an agile multiplexed cryptographic provider called [EverCrypt].
-As such, the full [HACL*] repository contains many software artifacts and a complicated build system
-that can be intimidating to a crypto developer who simply wishes to use verified crypto.
-
-This repository addresses this gap by presenting several usable crypto packages developed by Cryspen on top of HACL\*.
+This repository contains ready-to-use crypto packages developed by [Cryspen] on top of [HACL*].
 In particular, it contains a portable C crypto library that selects optimized implementations for each platform,
-as well as Rust, OCaml, and JavaScript bindings for this library. Cryspen is in the process of adding more usable APIs for crypto
-primitives, as well as extensive documentation for these APIs. Cryspen is also working on more optimized versions of some
-algorithms.
+as well as Rust, OCaml, and JavaScript bindings for this library.
+
+Currently, we are in the process of adding more usable APIs, [providing extensive documentation](https://tech.cryspen.com/hacl-packages/), and further optimizing the algorithms.
 
-## Build
+## What is HACL\* and why should you use HACL Packages?
 
-We uses [cmake] to configure the C build and [ninja] to build.
+[HACL*] is a collection of high-assurance cryptographic algorithms developed as part of [Project Everest].
+It includes source code written in [F*], generated C code, verified assembly code from the [Vale] project, and an agile multiplexed cryptographic provider called [EverCrypt].
+As such, the full HACL\* repository contains many software artifacts and a complicated build system that can be intimidating to a crypto developer who simply wishes to use verified crypto.
 
-Quick start: `./mach build --test`
+## Quickstart
 
-Build dependencies
+### Install build dependencies
 
+We need the following dependencies ...
+
+- [clang] (X.X or newer) // TODO
```

Comment:
Not needed. You can use GCC as well or whatever you prefer by setting `CC`
Please file an issue to use `cc` in `mach` instead of `clang`.

Reply:
Resolved in later commits.